README.md

```diff
@@ -626,9 +627,184 @@ class BookList extends React.Component {
 
 # Child nodes
 
-> TODO
-> ViewChild vs refs
-> ContentChild vs this.props.children
+### AngularJS
+In an [AngularJS component](https://docs.angularjs.org/guide/component), we have access to the child node by injecting [`$element`](https://docs.angularjs.org/api/ng/function/angular.element) to its controller.
+```js
+class TextInputCtrl {
+  constructor($scope, $element) {
+    // $element is passed to the constructor and stored in an instance variable.
+    this.$scope = $scope;
+    this.$element = $element;
+  }
+  
+  $postLink() {
+    // When the links are done, we can use the $element attribute.
+    // It has a jqLite wrapped instance of the DOM element,
+    // so we have access to some jQuery methods.
+    const input = this.$element.find('input');
+    input.on('change', console.log);
+  }
+}
+
+const component = {
+  controller: TextInputCtrl
+  template: `
+    <div>
+      <input type="text" />
+    </div>
+  `
+};
+
+angular.module('app', [])
+  .component('textInput', component);
+```
+
+### Angular
+Angular provides two ways to deal with child nodes: `ViewChild(ren)` and `ContentChild(ren)`. They both have the same purpose, but there are different use cases for them.
+
+#### ViewChild
+[`ViewChild`](https://angular.io/api/core/ViewChild) returns the specified element from the view DOM. [`ViewChildren`](https://angular.io/api/core/ViewChildren) does the same, but for multiple elements (returning as a `QueryList`).
+```ts
+import {
+  Component,
+  Directive,
+  ViewChild,
+  AfterViewInit
+} from '@angular/core';
+
+@Directive({ selector: 'child' })
```

Comment:
Could be `@Component` as well, right?

Reply:
Yeah, I just need to add the `template` or `templateUrl` metadata as well, according to the specification.